service/resource_router.go

```diff
@@ -0,0 +1,33 @@
+package service
+
+import (
+	"context"
+
+	"github.com/giantswarm/kvm-operator/service/keyv2"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+// NewResourceRouter determines which resources are enabled based upon the
+// version in the version bundle.
+func NewResourceRouter(resources []framework.Resource) func(ctx context.Context, obj interface{}) ([]framework.Resource, error) {
```

Comment:
I know this is a copy paste, but it doesn't need to be exported.

Reply:
sorry, but I don't get it (newbie go programmer). If I don't use uppercase then I can not use it in framework, right?

Reply:
ok got it. Since we are in same package it is not needed